src/site/stages/build/process-cms-exports/schemas/transformed/node-page.js

```diff
@@ -0,0 +1,56 @@
+// const { getFilter } = require('../../filters');
+
+module.exports = {
+  $id: 'node-page',
+  type: 'object',
+  properties: {
+    title: {
+      type: 'string',
+    },
+    fieldIntroText: {
+      type: 'string',
+    },
+    fieldDescription: {
+      type: ['string', 'null'],
+    },
+    fieldFeaturedContent: {
+      type: 'array',
+      items: {
+        $ref: 'Paragraph',
+      },
+    },
+    fieldContentBlock: {
+      type: 'array',
+      items: {
+        $ref: 'Paragraph',
+      },
+    },
+    fieldAlert: {
+      type: 'object',
+      properties: {
+        entity: {
+          type: ['null'],
+        },
+      },
+    },
+    fieldRelatedLinks: {
+      type: 'array',
+      items: {
+        $ref: 'paragraph-list_of_link_teasers',
+      },
+    },
+    fieldAdministration: {
+      type: 'array',
+      items: {
+        $ref: 'taxonomy_term-administration',
+      },
+    },
+    fieldPageLastBuilt: {
+      type: 'string',
+    },
+    entityMetaTags: {
+      $ref: 'MetaTags',
+    },
+  },
+  // required: getFilter('node-page'),
```

Comment:
We'll probably want to specify all the fields that are actually used in the templates.

Reply:
Standard key-value pairs for Node entities:
```
- `changed`
- `created`
- `default_langcode`
- `langcode`
- `menu_link`
- `moderation_state`
- `path`
- `promote`
- `revision_log`
- `revision_timestamp`
- `revision_translation_affected`
- `revision_uid`
- `status`
- `sticky`
- `title`
- `type`
- `uid`
- `uuid`
```

Key-value pairs for `node` `page`s:
```
  - `field_administration`
  - `field_alert`
  - `field_content_block`
  - `field_description`
  - `field_featured_content`
  - `field_intro_text`
  - `field_meta_tags`
  - `field_meta_title`
  - `field_page_last_built`
  - `field_plainlanguage_date`
  - `field_related_links`
```

From `src/site/stages/build/process-cms-exports/graphql-cms-export-differences/node.md` :)